api/envoy/extensions/filters/listener/tls_inspector/v3/tls_inspector.proto

```diff
@@ -17,4 +17,7 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 message TlsInspector {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.listener.tls_inspector.v2.TlsInspector";
+
+  // Populate `JA3` fingerprint hash using data from the TLS Client Hello packet.
+  bool enable_ja3_fingerprinting = 1;
```

Comment:
Is it possible that we'd want the default value of this to change in the future?  If so, please use `google.protobuf.BoolValue` instead of `bool`.

Reply:
Yeah, it's plausible that we may want to start using this for something internal, and that it would make sense to flip the default, so +1 on BoolValue type.

Reply:
/wait